src/common/NetworkManager.ts

```diff
@@ -376,6 +376,11 @@ export class NetworkManager extends EventEmitter {
     );
     this._requestIdToRequest.set(event.requestId, request);
     this.emit(NetworkManagerEmittedEvents.Request, request);
+    request.finalizeInterceptions().catch((error) => {
+      // This should never happen, but catch just in case
```

Comment:
comments are sentences

```suggestion
      // This should never happen, but catch just in case.
```